packages/common/leaders/components/index.marko

```diff
@@ -5,28 +5,31 @@ $ const {
   sectionAlias: alias,
   logoSrc: src,
   logoAlt: alt,
+  bodyText,
+  columns,
 } = input;
 
 <marko-web-query|{ node: leaders }| name="website-section" params={ alias, queryFragment }>
   $ const nodes = getAsArray(leaders, 'children.edges').map(({ node }) => node);
-  <marko-web-node-list modifiers=["nodes-flex-row"]>
+  <marko-web-node-list>
     <@header modifiers=["centered"]>
       <if(src)>
-        <img src=src alt=alt />
+        <img src=src alt=alt class="leaders__logo" />
       </if>
     </@header>
-    <@body modifiers=["centered"]>
-      Browse these leading suppliers:
-    </@body>
+    <if(bodyText)>
+      <@body modifiers=["centered"] value=bodyText />
+    </if>
     <@nodes nodes=nodes>
+      $ const colClass = columns === 2 ? 'col-lg-6 mb-block list-unstyled' : 'col list-unstyled';
       $ const mid = Math.floor(nodes.length / 2)
-      $ const groups = [
+      $ const groups = columns == 2 ? [
```

Comment:
You could make this automatic by dividing the nodes by the total number of columns. Same for the styling above (if you used a SASS mixin).